packages/block-editor/src/components/link-control/index.js

```diff
@@ -104,7 +109,10 @@ function LinkControl( {
 	const [ isEditingLink, setIsEditingLink ] = useState(
 		! value || ! value.url
 	);
+	const [ isResolvingLink, setIsResolvingLink ] = useState( false );
+	const [ errorMsg, setErrorMsg ] = useState( null );
```

Comment:
Minor: I would tend to encourage avoiding abbreviations altogether, as they rarely (if ever) improve the code, while leaving open potential for unclarity.

i.e.

```suggestion
	const [ errorMessage, setErrorMessage ] = useState( null );
```